lib/spoom/sorbet/translate/sorbet_sigs_to_rbs_comments.rb

```diff
@@ -94,6 +154,72 @@ def visit_attr(node)
           @last_sigs.clear
         end
 
+        #: (Prism::CallNode node) -> void
+        def visit_extend(node)
+          raise Error, "Expected extend" unless node.message == "extend"
+
+          return unless node.receiver.nil? || node.receiver.is_a?(Prism::SelfNode)
+          return unless node.arguments&.arguments&.size == 1
+
+          arg = node.arguments&.arguments&.first
+          return unless arg.is_a?(Prism::ConstantPathNode)
+          return unless arg.slice.match?(/^(::)?T::Helpers$/) || arg.slice.match?(/^(::)?T::Generic$/)
+
+          from = adjust_to_line_start(node.location.start_offset)
+          to = adjust_to_line_end(node.location.end_offset)
+
+          if to + 1 < @ruby_bytes.size && @ruby_bytes[to + 1] == "\n".ord
+            to += 1
+          end
+
+          @rewriter << Source::Delete.new(from, to)
+        end
+
+        #: (Prism::CallNode node) -> void
+        def visit_class_annotation(node)
+          unless node.message == "abstract!" || node.message == "interface!" || node.message == "sealed!" ||
+              node.message == "final!" || node.message == "requires_ancestor"
+            raise Error, "Expected abstract!, interface!, sealed!, final!, or requires_ancestor"
+          end
```

Comment:
Similar, I don't think we need this guard if we ourselves are the ones invoking this custom visit.